app/src/main/java/net/squanchy/schedule/filterschedule/TracksFilter.kt

```diff
@@ -0,0 +1,29 @@
+package net.squanchy.schedule.filterschedule
+
+import io.reactivex.Observable
+import io.reactivex.subjects.BehaviorSubject
+import net.squanchy.schedule.domain.view.Track
+import net.squanchy.service.repository.TracksRepository
+
+interface TracksFilter {
+
+    fun updateSelectedTracks(newSelectedTracks: Set<Track>)
+
+    val selectedTracks: Observable<Set<Track>>
+}
+
+private val selectedTracksSubject = BehaviorSubject.create<Set<Track>>()
```

Comment:
This is, for now, a top-level (and thus, singleton) property, but it shouldn't. See the description.